ovirt_imageio/_internal/handlers/tickets.py

```diff
@@ -91,17 +91,24 @@ def delete(self, req, resp, ticket_id):
         requests in case of network failures. See
         https://tools.ietf.org/html/rfc7231#section-4.2.2.
         """
-        log.info("[%s] REMOVE ticket=%s", req.client_addr, ticket_id)
-
         if ticket_id:
             try:
-                self.auth.remove(ticket_id)
-            except errors.TicketCancelTimeout as e:
-                # The ticket is still used by some connection so we cannot
-                # remove it. The caller can retry the call again when the
-                # number connections reach zero.
-                raise http.Error(http.CONFLICT, str(e))
+                ticket = self.auth.get(ticket_id)
+            except KeyError:
+                # Ticket was already deleted.
+                log.info("[%s] REMOVE transfer=(deleted)", req.client_addr)
```

Comment:
Don't we want to display the "ticket_id" that caused that message, i.e., to say that "ticket with that id it was already deleted"? After you removed log message that was on line 94, now the log will not include which ticket caused that duplicate remove.

Reply:
We can show the ticket id, since the ticket was removed and there is not transfer to show. I added this to created similar log to the case when we remove the ticket, but this is
not very useful message, so we should either make it a debug message use the old format

    REMOVE ticket=